packages/cli/src/workflow-execute-additional-data.ts

```diff
@@ -495,6 +495,11 @@ function hookFunctionsSave(): IWorkflowExecuteHooks {
 						retryOf: this.retryOf,
 					});
 
+					// When going into the waiting state, store the pushRef in the execution-data
```

Comment:
Nit - Ideally we should comment on the why rather than the what.

Reply:
This code will be deleted in CAT-251